index.js

```diff
@@ -122,7 +122,6 @@ client.on('message', async msg => {
 			return
 		}
 	}
-
```

Comment:
Add line back in

Reply:
the tack '-'?

Reply:
If I add it back in on line 125 it breaks everything. Why is it there and what is its purpose

Reply:
i am dumb